class/Core/Uninstall.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Uninstall class
+ *
+ * @package notification
+ */
+
+namespace BracketSpace\Notification\Core;
+
+use BracketSpace\Notification\Utils\Files;
+use BracketSpace\Notification\Utils\View;
+use BracketSpace\Notification\Admin\Extensions;
+
+/**
+ * Uninstall class
+ */
+class Uninstall {
+
+	/**
+	 * Uninstall constructor
+	 */
+	public function __construct() {
+
+		register_uninstall_hook(NOTIFICATION_FILE, ['BracketSpace\Notification\Core\Uninstall', 'remove_plugin_data']);
```

Comment:
There shouldn't be any actions in constructor. Please move it to the main plugin file, or the best to Runtime class

Reply:
Please check fix here:
https://github.com/BracketSpace/Notification/commit/77c96612999572852205560f715214c110ebb338